tasks/pkg-redhat.yml

```diff
@@ -39,9 +39,11 @@
   when: (datadog_agent_version != "") and (not ansible_check_mode)
   notify: restart datadog-agent
 
-# Using `yum` directly to work around an Ansible bug:
-# https://github.com/ansible/ansible/issues/20608
 - name: Install latest datadog-agent package
-  command: yum install -y datadog-agent
-  when: datadog_agent_version == ""
+  yum:
+    name: datadog-agent
+    update_cache: yes
+    state: latest
+  when:
+  - datadog_agent_version == ""
```

Comment:
This could got back to one line, and do you want `not ansible_check_mode` to be consistent with above

Reply:
Good point, we do it for Debian based distros: https://github.com/DataDog/ansible-datadog/blob/34e4275e079d71b3ea775ad2ac3d73e2361dd0cc/tasks/pkg-debian.yml#L61 but not for Red Hat nor Suse. We could add for both. I believe your branch does not include the `pkg-suse.yaml` file so you'll need to rebase on master to make that change (if you want?).

Thanks!